docs/faq-5-misc.txt

```diff
@@ -33,6 +33,12 @@ that came with the version of OpenSSL you are using. The pod format
 documentation is included in each OpenSSL distribution under the docs
 directory.
 
+* I need a FIPS validated offering
+
+Please see
+@@@https://www.openssl.org/docs/fips.html@@@; the OpenSSL project no longer
+does FIPS validation work.
```

Comment:
the OpenSSL project is no longer involved in private label validations nor adding platforms to the existing certificates

(to be consistent with the later statements and to be factually correct - the project is doing FIPS validation work as part of the next release)